packages/core/core/src/AssetGraph.js

```diff
@@ -69,6 +69,8 @@ const nodeFromEntryFile = (entry: string) => ({
 });
 
 export default class AssetGraph extends Graph<AssetGraphNode> {
+  static VERSION = Graph.VERSION + ':1';
```

Comment:
Yeah, manually updating this definitely seems error prone. I'd be more comfortable if we moved these structures to their own package(s) so that things like `lerna version` could automatically bump versions when things change. Seems like right now, it would be okay to just have one package for these structures.

Reply:
For now, let's just use the core version from package.json. Seems easiest to me. then we can determine a more granular way to share caches between parcel versions where possible.